### PR TITLE
[#2624] fix(open-api): add openapi description for fileset

### PR DESCRIPTION
**Pull request by coolderli**
### What changes were proposed in this pull request?
- fix some incorrect descriptions for fileset on open-API doc.

### Why are the changes needed?

Fix: #2624 

### Does this PR introduce _any_ user-facing change?

- no

### How was this patch tested?

- `./gradlew build`

### CODE REVIEW

**Comment by coolderli** [comment]
@LauraXia123 Please help review this. Thanks.

File: docs/open-api/filesets.yaml [hunk 1]
```diff
@@ -183,22 +183,23 @@ components:
       properties:
         name:
           type: string
-          description: The name of the fileset
+          description: The name of the fileset. Can not be empty.
         type:
           type: string
-          description: The type of the fileset
+          description: The type of the fileset. The value of type should be EXTERNAL or MANAGED
```
Comment (jerryshao) [inline docs/open-api/filesets.yaml:189]: Is it uppercase or lowcase?

Comment (coolderli) [reply docs/open-api/filesets.yaml:189]: I think it should be uppercase.
image

```
 Fileset fileset =
                TreeLockUtils.doWithTreeLock(
                    ident,
                    LockType.WRITE,
                    () ->
                        dispatcher.createFileset(
                            ident,
                            request.getComment(),
                            Optional.ofNullable(request.getType()).orElse(Fileset.Type.MANAGED),
                            request.getStorageLocation(),
                            request.getProperties()));
            return Utils.ok(new FilesetResponse(DTOConverters.toDTO(fileset)));
```

**Review by jerryshao — APPROVED** [review]

**Comment by jerryshao** [comment]
Thanks @coolderli for your work, merging to main.

**Pull request merged**